deluge/core/alertmanager.py

```diff
@@ -174,7 +174,7 @@ def handle_alerts(self):
                 d = task.deferLater(reactor, 0, handler, alert)
                 on_handler_timeout = partial(
                     self.on_delayed_call_timeout,
-                    handler=handler.__qualname__,
+                    handler=handler.__class__.__qualname__,
```

Comment:
This flat-out produces an error, because __qualname__ is an attribute that classes (not their instances) have.

Reply:
When does it raise an error? Handlers are functions or methods not class instances so `handle.__class__.__qualname__` does not make seem to make sense here?

Reply:
> When does it raise an error? Handlers are functions or methods not class instances so `handle.__class__.__qualname__` does not make seem to make sense here?

See this log, for an example:

https://github.com/Lord-Kamina/deluge/actions/runs/10050873417/job/27779525108

Reply:
Although now... reading that again, it might be an issue specific to running the tests.

Reply:
@cas-- I have reverted that change and implemented a proper fix in `conftest` now.